lib/season_stats.rb

```diff
@@ -4,8 +4,7 @@ def winningest_coach(season_id)
         get_all_games(season_id)
         load_coach_array
         load_coach_hash
-        coach_percentage_hash
-        @coach_percentages.sort_by {|k, v| -v}.first[0]
+        coach_percentage_hash.sort_by {|k, v| -v}.first[0]
     end
 
     def get_all_games(season_id)
```

Comment:
Could put #get_all_games method in the StatTracker class and give it the @all_season_games  as an attribute so the instance variable is in StatTracker instead of the module.

Reply:
yeah, that's a good thought. I ultimately thought it made sense in season stats module because any time you want to run a season-based statistic on something, you'd want to narrow your data to only the game data from that specific season (@all_season_games). I mainly just didn't wanna mess with any class files outside of the module (like stat_tracker) to assure we wouldn't encounter any merging problems come Monday morning

Reply:
it'll require a bit of refactoring at this point so up to you if you want to try

Reply:
True. There's going to be quite a bit of stuff to resolve and refactor if we attempt this.